src/views/Home.vue

```diff
@@ -314,50 +344,31 @@ export default {
     },
     openUploader () {
       this.files = []
+      this.uploadPhotos = []
       this.uploadCounter = 0
       this.show = true
-      document.getElementById('uploadPanel').innerHTML = ''
     },
-    generateUploadImgElement (fileObj) {
-      var file = fileObj.file
-      var id = fileObj.id
-      var reader = new FileReader()
-      reader.onload = (function (file, id) {
-        // debugger
-        return function (e) {
-          document.getElementById('uploadPanel').innerHTML +=
-            ['<div id="', id, '" style="display:inline-block;position:relative"><img id="img-', id, '" src="', e.target.result, '" class="upload-preview upload-preview-pending"/></div>'].join('')
-        }
-      })(file, id)
+    generateUploadPhotoElement (fileObj) {
+      const self = this
```

Comment:
i prefer me because self is a object inside window.
and me is shorter lol

Reply:
OK no problem ^_^